requirements/base.txt

```diff
@@ -6,7 +6,7 @@
 git+https://github.com/edx/XBlock.git@9c634481dfc85a17dcb3351ca232d7098a38e10e#egg=XBlock
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@9538ee8a971d04dc1cb05e88f6aa0c36b224455c#egg=edx-submissions
+git+https://github.com/edx/edx-submissions.git@34d6987c6a3b3bd10b3c61c6abdddcb239fd74af#egg=edx-submissions==0.1.0
```

Comment:
I'm going to create a new tag when my PR to edx-submissions gets merged, then update this accordingly.